test/functional/artefacts_controller_test.rb

```diff
@@ -152,6 +152,19 @@ class ArtefactsControllerTest < ActionController::TestCase
         get :new
         assert_select "li[class~=active] a[href=/artefacts/new]"
       end
+
+      should 'render the tags partial if publisher is not an untaggable app' do
+        Settings.expects(:untaggable_apps).returns(%w{ smartanswers testapp }).at_least(1)
```

Comment:
I would favour `stubs` over `expects` unless setting an expectation here is important (which it appears not). It is a small yet important semantic difference.